examples/validation_tests.ipynb

```diff
@@ -0,0 +1,558 @@
+    "\n",
+    "axs[1].tick_params(labelsize=14)\n",
+    "axs[0].tick_params(labelsize=1)\n",
+    "axs[0].legend(fontsize=12)\n",
+    "fig.tight_layout()\n",
+    "fig.savefig('Einasto_precision.pdf')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Compare all to Colossus"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure()\n",
+    "fig, axs = plt.subplots (2, sharex=True, gridspec_kw={'hspace': 0}, figsize=(10,8))\n",
+    "\n",
+    "axs[0].set_xscale ('log')\n",
+    "axs[0].set_yscale ('log')\n",
+    "axs[1].set_yscale ('log')\n",
+    "axs[1].set_xlabel('R (Mpc)')\n",
+    "axs[0].set_ylabel(r'$\\Sigma$ [M$_\\odot$  Mpc$^{-2}$]')\n",
+    "axs[1].set_ylabel(r'$\\Sigma_{\\mathrm{i}} (R) / \\Sigma_{\\mathrm{col}} (R) - 1$')\n",
+    "axs[0].plot (r, col_Sigma_nfw, '-', label = 'NFW - Col', color = 'lightcoral')\n",
+    "axs[0].plot (r, col_Sigma_ein, '-', label = 'EIN - Col', color = 'cadetblue')\n",
+    "axs[0].plot (r, col_Sigma_her, '-', label = 'HER - Col', color = 'tan')\n",
+    "\n",
+    "axs[1].plot (r, np.abs (nc_Sigma_nfw_ana / col_Sigma_nfw - 1.0), '-', label = \"NFW - NC\", linestyle='-', color='lightcoral')\n",
+    "axs[1].plot (r, np.abs (nc_Sigma_nfw_num / col_Sigma_nfw - 1.0), '-', label = \"NFW - NC (num.)\", linestyle='-', color='lightcoral')\n",
+    "axs[1].plot (r, np.abs (ccl_Sigma_nfw_ana / col_Sigma_nfw - 1.0), '-', label = \"NFW - CCL\", linestyle='--', color='lightcoral')\n",
+    "axs[1].plot (r, np.abs (ccl_Sigma_nfw_num / col_Sigma_nfw - 1.0), '-', label = \"NFW - CCL (num.)\", linestyle=':', color='lightcoral')\n",
+    "axs[1].plot (r, np.abs (ct_Sigma_nfw / col_Sigma_nfw - 1.0), '-', label = \"NFW - CT\", linestyle='-.', color='lightcoral')\n",
+    "axs[1].plot (r, np.abs (ct_Sigma_nfw_num / col_Sigma_nfw - 1.0), '-', label = \"NFW - CT (num)\", linestyle='-.', color='lightcoral')\n",
+    "axs[1].plot (r, np.abs (nc_Sigma_ein / col_Sigma_ein - 1.0), '-', label = \"EIN - NC\", color='cadetblue', linestyle='-')\n",
+    "axs[1].plot (r, np.abs (ccl_Sigma_ein / col_Sigma_ein - 1.0), '-', label = \"EIN - CCL\", color='cadetblue', linestyle='--')\n",
+    "axs[1].plot (r, np.abs (ct_Sigma_ein / col_Sigma_ein - 1.0), '-', label = \"EIN - CT\", color='cadetblue', linestyle='-.')\n",
+    "axs[1].plot (r, np.abs (nc_Sigma_her / col_Sigma_her - 1.0), '-', label = \"HER - NC\", color='tan', linestyle='-')\n",
+    "axs[1].plot (r, np.abs (ccl_Sigma_her / col_Sigma_her - 1.0), '-', label = \"HER - CCL\", color='tan', linestyle='--')\n",
+    "\n",
+    "axs[0].legend(loc='best')\n",
+    "axs[1].legend(loc='best')\n",
+    "axs[1].grid()\n",
+    "plt.show ()\n",
+    "#fig.savefig('Sigma_precision.pdf')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure()\n",
```

Comment:
can remove this line since the figure is declared in the next line with plt.subplots

Reply:
Done.